client/src/client/startup.py

```diff
@@ -41,31 +45,35 @@ def __init__(self) -> None:
     def cans_setup(self) -> None:
         """Create all necessary directories."""
         rmtree(path=self._home_dir, ignore_errors=True)
-        mkdir(self._home_dir, mode=0o700)
-        mkdir(self._keys_dir, mode=0o700)
-        mkdir(self._backups_dir, mode=0o700)
+        os.mkdir(self._home_dir, mode=0o700)
+        os.mkdir(self._keys_dir, mode=0o700)
+        os.mkdir(self.backups_dir, mode=0o700)
 
     def is_first_startup(self) -> bool:
         """Check if user's EC EcPemKeyPair exists."""
-        priv_key_exists = path.isfile(self.user_private_key_path)
-        crypto_account_exists = path.isfile(self.crypto_account_path)
+        priv_key_exists = os.path.isfile(self.user_private_key_path)
+        crypto_account_exists = os.path.isfile(self.crypto_account_path)
 
         return not (priv_key_exists and crypto_account_exists)
 
     def create_crypto_account(self, passphrase: str) -> Account:
         """Create a libolm Account() object and store it."""
         account = Account()
         pickled = account.pickle(passphrase)
-        with open(self.crypto_account_path, "wb") as fd:
-            fd.write(pickled)
+        fd = os.open(
+            path=self.crypto_account_path,
+            flags=os.O_CREAT | os.O_WRONLY,
+            mode=0o700,
+        )
+        os.write(fd, pickled)
         return account
 
     def load_crypto_account(self, passphrase: str) -> Account:
         """Load an existing libolm Account() object."""
-        with open(self.crypto_account_path, "rb") as fd:
-            pickled = fd.read()
-            account = Account.from_pickle(pickled, passphrase)
-            return account
+        fd = os.open(path=self.crypto_account_path, flags=os.O_RDONLY)
```

Comment:
What is wrong with io.open? Do we need low-level writes and reads? Was it changed to use the flags?

Reply:
It was changed to use `modes` when creating new files. 
In order to avoid misunderstandings with python function `open` I changed it everywhere